tpp/src/main/Member.js

```diff
@@ -345,6 +346,109 @@ export default class Member extends CoreMember {
         });
     }
 
+    /**
+     * Redeems a recurring transfer token.
+     *
+     * @param token - token to redeem. Can also be a tokenId
+     * @param amount - amount to redeem
+     * @param currency - currency to redeem
+     * @param description - optional transfer description
+     * @param destinations - transfer destinations
+     * @param refId - ID that will be set on created Recurring Transfer
+     *                Token uses this to detect duplicates
+     *                caller might use this to recognize the transfer
+     *                if param empty, transfer will have random refId
+     * @return Recurring Transfer created as a result of this redeem call
+     */
+    redeemRecurringToken(
```

Comment:
in java it only takes `tokenId`, we should mimic that

Reply:
https://github.com/tokenio/sdk-java/blob/b90a866133f094c37597189dbc8b287872978333/tpp/src/main/java/io/token/tpp/Member.java#L240

The Java version has constructors that seems to account for all scenarios (number of params included in addition to tokenId). Can I make it so that they are optional, as denoted with the flow syntax with (?) ?